### DISCUSSION TITLE
Not really an issue, but ...

### DISCUSSION THREAD

**Discussion by BurdetteLamar** [ruby/rdoc#1203] [General]
I see that several contributors here will be at RubyConf in Chicago this week.  Should we be planning to get together?

**Comment by st0012** [comment]
I'll be there but haven't had any concrete plan yet. I'll be happy to have a drink together so feel free to message me on social media platforms (links in my profile) 😄

**Reply by BurdetteLamar** [reply]
Okay, thanks.

**Reply by st0012** [reply]
I'll also be at the ballroom during hackday. I'm wearing a penn state jacket.

**Comment by BurdetteLamar** [comment]
Will be there in a few minutes:  red shirt, black tie, gray tieback.

    On Thursday, November 14, 2024 at 11:13:57 AM CST, Stan Lo ***@***.***> wrote:  
 
 


I'll also be at the ballroom during hackday. I'm wearing a penn state jacket.

**Comment by st0012** [comment]
Since RubyConf has finished and we did meet there, I'm closing this as resolved 😄
